packages/core/src/actions/send.ts

```diff
@@ -123,10 +123,12 @@ function execute(
   const { to, event } = params;
 
   actorContext.defer(() => {
```

Comment:
I wonder if we could move this concern to the `system.sendTo`. It would be nice if we didn't have to worry about this here at all and if all sends would be processed in the very same way. That's outside of the scope of this PR though

Reply:
Probably not easily at the moment; something we can revisit